index.html

```diff
@@ -0,0 +1,184 @@
+                        <i class="fas fa-fire fa-4x"></i>
+                        <h3>Team Exercise</h3>
+                        <h6>4:00 p.m - 6:00 p.m</h6>
+                    </div>
+                </div>
+            </div>
+            <div class="row mt-4">
+                <div class="col-md-12 text-center">
+                    <button id="showProgramButton" class="btn btn-primary">See the Whole Program</button>
+                </div>
+            </div>
+            <div id="programCards" class="row mt-4">
+                <div class="col-md-4 mt-4">
+                    <div class="program-card card">
+                        <i class="fas fa-chalkboard-teacher fa-4x"></i>
+                        <h3>Additional Lecture</h3>
+                        <h5>Announcement soon</h5>
+                    </div>
+                </div>
+                <div class="col-md-4 mt-4">
+                    <div class="program-card card">
+                        <i class="fas fa-museum fa-4x"></i>
+                        <h3>Additional Exhibition</h3>
+                        <h6>Announcement soon</h6>
+                    </div>
+                </div>
+                <div class="col-md-4 mt-4">
+                    <div class="program-card card">
+                        <i class="fas fa-comments fa-4x"></i>
+                        <h3>Additional Forum</h3>
+                        <h6>Announcement soon</h6>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="speakers" class="about">
+        <h2>Featured Speakers</h2>
+        <p>Our API Festival brings together some of the brightest minds in the industry. Learn from industry experts,
+            thought leaders, and innovators who have pioneered the world of APIs.</p>
+        <p>Below is the list of our esteemed lineup of speakers.</p>
+        <div class="container mt-4">
+            <h2>Speakers List</h2>
+            <div class="row" id="speakerContainer">
+            </div>
+            <div id="seeMoreButton">
+                <button class="btn btn-primary" onclick="showHiddenSpeakers()">See More</button>
+            </div>
+        </div>
+    </section>
+    <section id="tickets" class="about">
+        <h2>Get Your Tickets</h2>
+        <p>Don't miss out on this incredible opportunity to immerse yourself in the world of APIs. Secure your tickets
+            today to guarantee your spot at the API Festival.</p>
+        <p>Early bird ticket sales will begin soon. Stay tuned for updates!</p>
+    </section>
+
+    <section id="partners" class="about">
+        <h2>Partners</h2>
+        <p>Here is a list of our wonderful partners who sponsored this event</p>
+        <div class="partners-container">
+            <img src="Images/6.png" alt="Company 1" class="partner-logo">
+            <img src="Images/7.png" alt="Company 2" class="partner-logo">
+        </div>
+    </section>
```

Comment:
- [ ] Please make sure your partner's section on the Home page matches the Behance design as well:
- Behance design:
  ![image](https://github.com/NgachaIan/Ngacha_Portfolio/assets/104968236/664989ed-40f4-4d6d-a5d3-980dfe2d61fa)
- Your design:
  ![image](https://github.com/NgachaIan/Ngacha_Portfolio/assets/104968236/71e034ee-523d-4569-8c5a-bfd23b95cffd)